### PR TITLE
Update Set-CsTeamsGuestMessagingConfiguration.md

### PR DESCRIPTION
**Pull request by munindersingh**
Add "AllowUserDeleteChat" Parameter

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md [hunk 1]
```diff
@@ -166,6 +166,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowUserDeleteChat
+Turn this setting on to allow users to permanently delete their 1:1, group chat, and meeting chat as participants (this deletes the chat only for them, not other users in the chat). Set this to TRUE to allow. Set this to FALSE to prohibit.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md:170]:
```suggestion
Turn this setting on to allow users to permanently delete their one-on-one chat, group chat, and meeting chat as participants (this deletes the chat only for them, not other users in the chat). Set this to TRUE to allow. Set this to FALSE to prohibit.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @munindersingh, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md [hunk 2]
```diff
@@ -21,15 +21,15 @@ TeamsGuestMessagingConfiguration determines the messaging settings for the guest
 ```
 Set-CsTeamsGuestMessagingConfiguration [-Tenant <Guid>] [-AllowUserEditMessage <Boolean>]
  [-AllowImmersiveReader <Boolean>] [-AllowUserDeleteMessage <Boolean>] [-AllowUserChat <Boolean>] [-AllowGiphy <Boolean>]
- [-GiphyRatingType <String>] [-AllowMemes <Boolean>] [-AllowStickers <Boolean>] [[-Identity] <XdsIdentity>]
+ [-AllowUserDeletChat <Boolean>] [-GiphyRatingType <String>] [-AllowMemes <Boolean>] [-AllowStickers <Boolean>] [[-Identity] <XdsIdentity>]
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md:24]:
```suggestion
 [-AllowUserDeleteChat <Boolean>] [-GiphyRatingType <String>] [-AllowMemes <Boolean>] [-AllowStickers <Boolean>] [[-Identity] <XdsIdentity>]
```

File: skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md [hunk 3]
```diff
@@ -21,15 +21,15 @@ TeamsGuestMessagingConfiguration determines the messaging settings for the guest
...
  [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### Instance
 ```
 Set-CsTeamsGuestMessagingConfiguration [-Tenant <Guid>] [-AllowUserEditMessage <Boolean>]
  [-AllowImmersiveReader <Boolean>] [-AllowUserDeleteMessage <Boolean>] [-AllowUserChat <Boolean>] [-AllowGiphy <Boolean>]
- [-GiphyRatingType <String>] [-AllowMemes <Boolean>] [-AllowStickers <Boolean>] [-Instance <PSObject>] [-Force]
+ [-AllowUserDeletChat <Boolean>] [-GiphyRatingType <String>] [-AllowMemes <Boolean>] [-AllowStickers <Boolean>] [-Instance <PSObject>] [-Force]
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md:32]:
```suggestion
 [-AllowUserDeleteChat <Boolean>] [-GiphyRatingType <String>] [-AllowMemes <Boolean>] [-AllowStickers <Boolean>] [-Instance <PSObject>] [-Force]
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@munindersingh Gentle reminder. Thanks

**Comment by yogkumgit** [comment]
@munindersingh Please implement the suggested changes, else we will proceed to close this PR. Thanks

File: skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md (hunk 2 above)
Comment (dariomws) [reply skype/skype-ps/skype/Set-CsTeamsGuestMessagingConfiguration.md:24]: @munindersingh Please also implement the suggested changes. Thanks!

**Review by munindersingh — COMMENTED** [review]
changes made -AllowUserDeleteChat

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**